static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-rebaseline-support-proposition.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+import template from 'lodash/utilities/template';
+import singleButtonTemplate from 'raw-loader!common/views/acquisitions-epic-single-button.html';
+import config from 'lib/config';
+
+const buildButtonTemplate = ({ supportUrl }) =>
+    template(singleButtonTemplate, {
+        url: supportUrl,
+    });
+
+export const acquisitionsEpicRebaselineSupportProposition = makeABTest({
+    id: 'AcquisitionsEpicRebaselineSupportProposition',
+    campaignId: 'sandc_epic_rebaseline_support_proposition',
+
+    start: '2017-07-05',
+    expiry: '2017-08-05',
+
+    author: 'Ap0c',
+    description:
+        'This creates a single-button version of the epic that links off to the new support frontend bundles landing page',
+    successMeasure: 'Conversion rate',
+    idealOutcome:
+        'We get a baseline for conversion of the bundles landing page',
+
+    audienceCriteria: 'UK all devices',
+    audience: 0,
```

Comment:
i'd advise against that because of @davidfurey's point - you should release it with the audience size you want for the test, you can always check it on code if local testing isn't enough.